sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/impl/RestServiceServer.java

```diff
@@ -218,4 +229,15 @@ private void enableCors(Environment environment) {
   protected void bootstrapLogging() {
     // disable dropwizard logging, it will use external logback
   }
+
+  private void registerJvmMetrics(MetricRegistry registry) {
+    registry.register("jvm.attribute", new JvmAttributeGaugeSet());
```

Comment:
Should these metrics be at the top-level like they are in health checker? 

https://github.com/stargate/stargate/blob/v2.0.0-ALPHA-14/health-checker/src/main/java/io/stargate/health/Server.java#L103-L105

Reply:
I also removed: